<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C5CashDoc</class>
 <widget class="QWidget" name="C5CashDoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Document number</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="6">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="2" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Total</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="C5LineEdit" name="leTotal">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="C5TableWidget" name="tbl">
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Amount</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QToolButton" name="btnNewRow">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/res.qrc">
       <normaloff>:/add.png</normaloff>:/add.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="C5LineEditWithSelector" name="leOutput">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Cash input</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Cash, output</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QToolButton" name="btnOpenStoreDoc">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../resources/res.qrc">
       <normaloff>:/folder.png</normaloff>:/folder.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="C5DateEdit" name="deDate">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="C5LineEditWithSelector" name="leInput">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QLineEdit" name="leOutputName">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="C5LineEditWithSelector" name="leInputName">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Remarks</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="C5LineEdit" name="leDocNum">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="5">
    <widget class="QLineEdit" name="leRemarks"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lbStoreDoc">
     <property name="text">
      <string>Store document</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="leStoreDoc">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Partner</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="C5LineEditWithSelector" name="lePartner">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="C5LineEditWithSelector" name="lePartnerName">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEditWithSelector</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineeditwithselector.h</header>
  </customwidget>
  <customwidget>
   <class>C5LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>C5TableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">c5tablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>C5DateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">c5dateedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
